source/parsing/Preprocessor.cpp

```diff
@@ -435,6 +435,10 @@ Token Preprocessor::nextRaw() {
         }
     }
 
+    // Ensure EoF at the end of trivia to prevent inlining issues
```

Comment:
The code is write but the comment is misleading; this is is EndOfLine (EoL) not EndOfFile (EoF)